### PR TITLE
Add declaration of parameters in ROSPid.

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

File: src/pid_ros.cpp [hunk 1]
```diff
@@ -126,7 +129,7 @@ PidROS::getDoubleParam(const std::string & param_name, double & value)
 bool
 PidROS::initPid()
 {
-  double p, i, d, i_min, i_max;
+  double p, i, d, i_min, i_max = std::numeric_limits<double>::quiet_NaN();
```
Comment (bmagyar) [inline src/pid_ros.cpp:132]: this only sets the value for `i_max` which I'm sure is not what you intended.

For proof see https://godbolt.org/z/vW8vx3r3j

Comment (destogl) [reply src/pid_ros.cpp:132]: Thanks for saving the day!

Comment (destogl) [reply src/pid_ros.cpp:132]:
```suggestion
  double p, i, d, i_min, i_max;
  p = i = d = i_min = i_max = std::numeric_limits<double>::quiet_NaN();
```

File: test/pid_parameters_tests.cpp [hunk 2]
```diff
@@ -205,23 +205,23 @@ TEST(PidParametersTest, GetParametersFromParams)
...
-
-  rclcpp::Parameter param;
+  node->set_parameter(rclcpp::Parameter("p", P));
+  node->set_parameter(rclcpp::Parameter("i", I));
+  node->set_parameter(rclcpp::Parameter("d", D));
+  node->set_parameter(rclcpp::Parameter("i_clamp_max", I_MAX));
+  node->set_parameter(rclcpp::Parameter("i_clamp_min", I_MIN));
 
   ASSERT_TRUE(node->get_parameter("p", param));
   ASSERT_EQ(param.get_value<double>(), P);
```
Comment (bijoua29) [inline test/pid_parameters_tests.cpp:227]: This part of the test is just testing Node parameter functionality - nothing to do with PidROS testing

**Review by bijoua29 — APPROVED** [review]

**Review by bmagyar — APPROVED** [review]

File: test/pid_parameters_tests.cpp (hunk 2 above)
Comment (bmagyar) [reply test/pid_parameters_tests.cpp:227]: Agreed

**Review by bmagyar — APPROVED** [review]

**Pull request merged**